src/components/splashScreen/SplashScreen.tsx

```diff
@@ -0,0 +1,73 @@
+import {useEffect, useRef} from 'react';
+import {Animated, View} from 'react-native';
+
+import BaseContainer from '@components/baseContainer';
+import {IMAGES} from '@constants/imageConstants';
+import useScalingMetrics from '@hooks/useScalingMetrics';
+
+import {splashStyles} from './styles';
+
+const SplashScreen = () => {
+  const value = useRef(new Animated.Value(0)).current;
+  const scale = useRef(new Animated.Value(0)).current;
+  const fadeAnim = useRef(new Animated.Value(0)).current; // To control fade-in of the title
+  const translateY = useRef(new Animated.Value(0)).current;
+  const styles = splashStyles(scale, value, translateY, fadeAnim);
+  const {scaleSize} = useScalingMetrics();
+
+  const animate = () => {
```

Comment:
can define this function outside the component passing the animation references as agrs

this will make the code cleaner